aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -373,6 +373,13 @@ struct Vec256 {
     }
     return ret;
   }
+  Vec256<T> fmod(const Vec256<T> &exp) const {
```

Comment:
super nit: `exp` is terrible arg name =)

Reply:
thank you.  rename arg name to `b`